helper/storagepacker/storagepacker_v1.go

```diff
@@ -17,48 +17,48 @@ import (
 )
 
 const (
-	bucketCount                = 256
-	StoragePackerBucketsPrefix = "packer/buckets/"
+	bucketCount                       = 256
+	DefaultStoragePackerBucketsPrefix = "packer/buckets/"
```

Comment:
Perhaps add a comment for this value to indicate that this prefix is used for entity.